pkg/controller/networkpolicy/validate.go

```diff
@@ -390,3 +459,31 @@ func (t *tierValidator) deleteValidate(oldObj interface{}, userInfo authenticati
 	}
 	return "", true
 }
+
+// validateAntreaGroupSelectors ensures that an IPBlock is not set along with namespaceSelector and/or a
+// podSelector.
+func validateAntreaGroupSelectors(s corev1a2.GroupSpec) (string, bool) {
+	if s.IPBlock != nil {
```

Comment:
Dumb question - so such validation cannot be done with OpenAPI?

Reply:
generally they can be.. but i did not find a way to validate this combination 
i.e. allow podSelector OR namespaceSelector OR podSelector & namespaceSelector OR ipBlock only
i tried the following which did not work as expected
```
oneOf:
  - required: ["podSelector"]
  - required: ["namespaceSelector"]
  - required: ["podSelector", "namespaceSelector"]
  - required: ["ipBlock"]
```

Reply:
Ok.